e2e/libs/backupstore/s3.py

```diff
@@ -24,70 +35,87 @@ def get_api_client(self, client, core_api, minio_secret_name):
+
+        minio_cert_file_path = "/e2e/minio_cert.crt"
+        with open(minio_cert_file_path, 'w') as minio_cert_file:
+            base64_minio_cert = \
+                base64.b64decode(base64_minio_cert).decode("utf-8")
+            minio_cert_file.write(base64_minio_cert)
+
+        os.environ["SSL_CERT_FILE"] = minio_cert_file_path
 
         return Minio(minio_endpoint_url,
                      access_key=minio_access_key,
                      secret_key=minio_secret_key,
                      secure=True)
 
-    def get_backupstore_bucket_name(self, client):
+    def get_backupstore_bucket_name(self):
         backupstore = self.get_backup_target()
         assert self.is_backupTarget_s3(backupstore)
         bucket_name = urlparse(backupstore).netloc.split('@')[0]
         return bucket_name
 
-    def get_backupstore_path(self, client):
+    def get_backupstore_path(self):
         backupstore = self.get_backup_target()
         assert self.is_backupTarget_s3(backupstore)
         backupstore_path = urlparse(backupstore).path.split('$')[0].strip("/")
         return backupstore_path
 
-    def get_backup_volume_prefix(self, client, volume_name):
+    def get_backup_volume_prefix(self, volume_name):
         backupstore_bv_path = self.backup_volume_path(volume_name)
-        backupstore_path = self.get_backupstore_path(client)
+        backupstore_path = self.get_backupstore_path()
         return backupstore_path + backupstore_bv_path
 
-    def get_backup_cfg_file_path(self, client, volume_name, backup_name):
-        prefix = self.get_backup_volume_prefix(client, volume_name)
+    def get_backup_cfg_file_path(self, volume_name, backup_name):
+        prefix = self.get_backup_volume_prefix(volume_name)
         return prefix + "/backups/backup_" + backup_name + ".cfg"
 
-    def get_volume_cfg_file_path(self, client, volume_name):
-        prefix = self.get_backup_volume_prefix(client, volume_name)
+    def get_volume_cfg_file_path(self, volume_name):
+        prefix = self.get_backup_volume_prefix(volume_name)
         return prefix + "/volume.cfg"
 
-    def get_backup_blocks_dir(self, client, volume_name):
-        prefix = self.get_backup_volume_prefix(client, volume_name)
+    def get_backup_blocks_dir(self, volume_name):
+        prefix = self.get_backup_volume_prefix(volume_name)
         return prefix + "/blocks"
 
-    def create_file_in_backupstore(self, client, core_api, file_path, data={}): # NOQA
+    def create_file_in_backupstore(self, file_path, data={}): # NOQA
+
+        process = self.port_forward()
 
         secret_name = self.get_secret()
 
-        minio_api = self.get_api_client(client, core_api, secret_name)
-        bucket_name = self.get_backupstore_bucket_name(client)
+        minio_api = self.get_api_client(secret_name)
+        bucket_name = self.get_backupstore_bucket_name()
 
         if len(data) == 0:
             data = {"testkey": "test data from mino_create_file_in_backupstore()"}
```

Comment:
typo in `mino_create_file_in_backupstore`

Reply:
Refined. Thank you!